### DISCUSSION TITLE
Looking for a Dev or Dev Team with experience and success in launching FluidFramework platforms

### DISCUSSION THREAD

**Discussion by joesalvati68** [microsoft/FluidFramework#13877] [Q&A]
Small tech start-up here. Growing quickly and looking for a Platform environment to help us distinguish ourselves.  Would like to discuss a FluidFramework platform with a dev or devs who have experience in building and launching.

I can be reached at joe@arcuity.com 

Thanks in advance.

**Comment by bannarisoftwares** [comment]
Still looking @joesalvati68 , i m interested
